engine/src/cmd/base_util.cpp

```diff
@@ -670,6 +670,7 @@ void CompPython(int room,
             EnumMap::Pair("News", BaseComputer::NEWS),
             EnumMap::Pair("Info", BaseComputer::INFO),
             EnumMap::Pair("LoadSave", BaseComputer::LOADSAVE),
+            EnumMap::Pair("Network", BaseComputer::NETWORK),
```

Comment:
It appears that we have to keep this until we do away with the corresponding Main Menu item.